src/as/aswebglue.spec.ts

```diff
@@ -10,5 +10,13 @@ const num: u32 = SHORT * 2
 describe('aswebglue', () => {
 	it('was imported', () => {
 		expect(num).toBe(SHORT * 2)
+
+		const gl = new WebGLRenderingContext('canvas', 'webgl2')
+
+		expect(gl instanceof WebGLRenderingContext).toBe(true)
```

Comment:
It works! Now onto porting Three WebGLRenderer stuff...

Reply:
```ts
expect(gl instanceof WebGLRenderingContext).toBeTruthy()
```

This particular testing function is great for references too, asserting they aren't null.